indico/modules/events/registration/blueprint.py

```diff
@@ -222,6 +222,7 @@
 
 # API
 _bp.add_url_rule('/api/registration-forms', 'api_registration_forms', api_misc.RHAPIRegistrationForms)
+_bp.add_url_rule('/api/timetable', 'api_event_timetable', api_misc.RHAPIEventTimeTable, methods=('GET',))
```

Comment:
```suggestion
_bp.add_url_rule('/api/timetable', 'api_event_timetable', api_misc.RHAPIEventTimeTable)
```

GET is already the default